### DISCUSSION TITLE
SGS01 moisture sensor

### DISCUSSION THREAD

**Discussion by joseruisa** [Koenkk/zigbee2mqtt#23664] [Devices]
Hi there,

Device indicated not supported by Z2M

Any way to add correctly the device?

<img width="936" alt="water" src="https://github.com/user-attachments/assets/71c1e21c-7548-47ff-9746-c46d96dde606">

![soil](https://github.com/user-attachments/assets/79808610-d76c-4f21-ab40-b72caeef2ba0)

**Comment by mwitkow** [comment]
FWIW, I've got a similar device and the `generated_external_definition` is:

```

const definition = {
    zigbeeModel: ['TS0601'],
    model: 'TS0601',
    vendor: '_TZE284_aao3yzhs',
    description: 'Automatically generated definition',
    extend: [],
    meta: {},
};
```

It seems it's already in `zigbee-herdman-converters` 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/7815

I don't know how often these get picked up. Perhaps it's a question of running the latest zigbee2mqtt?

**Comment by Koenkk** [comment]
`_TZE284_aao3yzhs` will be supported in the 1 September release

**Reply by joseruisa** [reply]
Thanks for the share
